app/services/user_service.py

```
@@ -0,0 +1,7 @@
+# services/user_service.py
+from crud.user import get_user
```

Ruff (0.8.2)

`crud.user.get_user` imported but unused

Remove unused import: `crud.user.get_user`

(F401)